### PR TITLE
[excel] (Samples) Adjust how the samples and scenarios are branded

### PR DESCRIPTION
**Pull request by AlexJerabek**
This PR adjusts the sample list and TOC to better delineate between types of samples. Most significantly, the Power Automate samples are separated so they're easier to find.

### CODE REVIEW

File: docs/resources/samples/range-samples.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
-title: Ranges in Office Scripts
-description: A collection of code samples showing how to work with Excel ranges in Office Scripts.
-ms.date: 09/08/2023
+title: 'Ranges: work with the grid in Office Scripts'
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/samples/range-samples.md:2]:
```suggestion
title: 'Ranges: Work with the grid in Office Scripts'
```

File: docs/resources/samples/range-samples.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+description: A collection of code samples showing how to work with Excel ranges, collections of cells, in Office Scripts.
+ms.date: 12/05/2023
 ms.localizationpriority: medium
 ---
 
-# Ranges in Office Scripts
+# Ranges: work with the grid in Office Scripts
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/samples/range-samples.md:8]:
```suggestion
# Ranges: Work with the grid in Office Scripts
```

File: docs/resources/samples/samples-overview.md [hunk 3]
```diff
@@ -21,38 +22,45 @@ This section contains [Office Scripts](../../overview/excel.md) based solutions
...
-| [Add images to a workbook](add-image-to-workbook.md) | This sample adds an image to a workbook and copies an image across sheets.|
-| [Copy multiple Excel tables into a single table](copy-tables-combine.md) | This sample combines data from multiple Excel tables into a single table that includes all the rows. |
-| [Data validation: dropdown lists, prompts, and warning pop-ups](data-validation-samples.md) | These samples show how to use data validation to mandate specific conditions for cell data and how the user is alerted to these rules. |
-| [Create a workbook table of contents](table-of-contents.md) | This sample creates a table of contents with links to each worksheet. |
-| [JavaScript `Date` samples](javascript-dates.md) | A collection of samples that show how to translate between JavaScript and Excel date formats. |
-| [Record day-to-day changes in Excel and report them with a Power Automate flow](report-day-to-day-changes.md) | This sample uses a scheduled Power Automate flow to record daily readings and report the changes. |
+| [Ranges: work with the grid](range-samples.md) | These samples show how to work the `Range` object, which is central to most scripts. |
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/samples/samples-overview.md:25]:
```suggestion
| [Ranges: Work with the grid](range-samples.md) | These samples show how to work the `Range` object, which is central to most scripts. |
```

File: docs/resources/samples/samples-overview.md [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
 ---
 
 # Office Scripts samples and scenarios
 
 This section contains [Office Scripts](../../overview/excel.md) based solutions that help end users achieve automation of daily tasks. It contains realistic scenarios that users face and provides detailed solutions.
 
-- [Basics](#basics) showcase common examples that make up larger scripts. Most don't need a specific workbook or dataset and can be run in the workbook of your choice.
-- [Beyond the basics](#beyond-the-basics) are samples that are more involved or solve a particular problem. Some use Power Automate with an Office Script as a integral part of the flow.
-- [Scenarios](#scenarios) are a few larger samples that demonstrate real-world use cases.
+- [Basics](#basics) showcase common examples that make up larger scripts. They are categorized based on a common Excel feature.
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/samples/samples-overview.md:12]:
```suggestion
- [Basics](#basics) showcase common examples that make up larger scripts. They're categorized based on a common Excel feature.
```

File: docs/resources/samples/samples-overview.md [hunk 5]
```diff
@@ -21,38 +22,45 @@ This section contains [Office Scripts](../../overview/excel.md) based solutions
...
+| [Cross-reference workbooks](excel-cross-reference.md) | This sample uses Office Scripts and Power Automate to cross-reference and validate information in different workbooks. |
+| [Email chart and table images](email-images-chart-table.md) | This sample uses Office Scripts and Power Automate actions to create a chart and send that chart as an image by email. |
+| [Record day-to-day changes in Excel and report them with a flow](report-day-to-day-changes.md) | This sample uses a scheduled Power Automate flow to record daily readings and report the changes. |
+| [Run a script on all Excel files in a folder](automate-tasks-on-all-excel-files-in-folder.md) | This project performs a set of automation tasks on all files situated in a folder on OneDrive for Business (can also be used for a SharePoint folder). It performs calculations on the Excel files, adds formatting, and inserts a comment that @mentions a colleague. |
 | [Write a large dataset](write-large-dataset.md) | This sample shows how to send a large range as smaller subranges. |
```
Comment (michelleranmsft) [inline docs/resources/samples/samples-overview.md:61]: At first glance, I'm not sure it's clear what "send" means and how this scenario fits in the cross-application section?

File: docs/toc.yml [hunk 6]
```diff
@@ -66,57 +66,59 @@ items:
       href: resources/samples/samples-overview.md
     - name: Basics
       items:
-      - name: Ranges
+      - name: 'Ranges: work with the grid'
         href: resources/samples/range-samples.md
-      - name: Tables
-        href: resources/samples/table-samples.md
+      - name: "Data validation: dropdown lists, prompts, and warning pop-ups"
```
Comment (ElizabethSamuel-MSFT) [inline docs/toc.yml:71]:
```suggestion
      - name: 'Data validation: Dropdown lists, prompts, and warning pop-ups'
```

File: docs/resources/samples/samples-overview.md [hunk 7]
```diff
@@ -21,38 +22,45 @@ This section contains [Office Scripts](../../overview/excel.md) based solutions
...
+| [Convert CSV files to Excel workbooks](convert-csv.md) | This sample uses Office Scripts and Power Automate to create .xlsx files from .csv files. |
+| [Cross-reference workbooks](excel-cross-reference.md) | This sample uses Office Scripts and Power Automate to cross-reference and validate information in different workbooks. |
+| [Email chart and table images](email-images-chart-table.md) | This sample uses Office Scripts and Power Automate actions to create a chart and send that chart as an image by email. |
+| [Record day-to-day changes in Excel and report them with a flow](report-day-to-day-changes.md) | This sample uses a scheduled Power Automate flow to record daily readings and report the changes. |
+| [Run a script on all Excel files in a folder](automate-tasks-on-all-excel-files-in-folder.md) | This project performs a set of automation tasks on all files situated in a folder on OneDrive for Business (can also be used for a SharePoint folder). It performs calculations on the Excel files, adds formatting, and inserts a comment that @mentions a colleague. |
```
Comment (michelleranmsft) [inline docs/resources/samples/samples-overview.md:60]: "This sample" instead of "this project" to be consistent?

File: docs/toc.yml [hunk 8]
```diff
@@ -66,57 +66,59 @@ items:
       href: resources/samples/samples-overview.md
     - name: Basics
       items:
-      - name: Ranges
+      - name: 'Ranges: work with the grid'
```
Comment (ElizabethSamuel-MSFT) [inline docs/toc.yml:69]:
```suggestion
      - name: 'Ranges: Work with the grid'
```

File: docs/resources/samples/samples-overview.md [hunk 9]
```diff
@@ -21,38 +22,45 @@ This section contains [Office Scripts](../../overview/excel.md) based solutions
...
-| [Data validation: dropdown lists, prompts, and warning pop-ups](data-validation-samples.md) | These samples show how to use data validation to mandate specific conditions for cell data and how the user is alerted to these rules. |
-| [Create a workbook table of contents](table-of-contents.md) | This sample creates a table of contents with links to each worksheet. |
-| [JavaScript `Date` samples](javascript-dates.md) | A collection of samples that show how to translate between JavaScript and Excel date formats. |
-| [Record day-to-day changes in Excel and report them with a Power Automate flow](report-day-to-day-changes.md) | This sample uses a scheduled Power Automate flow to record daily readings and report the changes. |
+| [Ranges: work with the grid](range-samples.md) | These samples show how to work the `Range` object, which is central to most scripts. |
+| [Data validation samples: dropdown lists, prompts, and warning pop-ups](data-validation-samples.md) | These samples show how to use data validation to mandate specific conditions for cell data and how the user is alerted to these rules. |
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/samples/samples-overview.md:26]:
```suggestion
| [Data validation samples: Dropdown lists, prompts, and warning pop-ups](data-validation-samples.md) | These samples show how to use data validation to mandate specific conditions for cell data and how the user is alerted to these rules. |
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
My suggestions are how I interpreted what the [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28754) says. Not sure about the list-y ones (the "dropdown lists, .." ones).

File: docs/resources/samples/samples-overview.md (hunk 5 above)
Comment (AlexJerabek) [reply docs/resources/samples/samples-overview.md:61]: Fair point. I'll clarify the scenario.

> "This sample shows how to work with a large range as smaller subranges. This helps flows handle data transfer limits and operation timeouts.

**Pull request merged**